src/Durable/SetFunctionInvocationContext.cs

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#pragma warning disable 1591 // Missing XML comment for publicly visible type or member 'member'
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
+{
+    using System.Collections;
+    using System.Management.Automation;
+
+    /// <summary>
+    /// Set the orchestration context.
+    /// </summary>
+    [Cmdlet("Set", "FunctionInvocationContext")]
+    public class SetFunctionInvocationContextCommand : PSCmdlet
+    {
+        internal const string ContextKey = "OrchestrationContext";

```

Comment:
Are you making this internal so it can be access anywhere within the assembly?

Reply:
Yes, and it is actually used in `InvokeActivityFunctionCommand`

Reply:
Great, thanks.

Reply:
Ok, resolving conversation.